.eslintrc.js

```diff
@@ -34,6 +40,12 @@ module.exports = {
       rules: {
         '@typescript-eslint/no-var-requires': 'off'
```

Comment:
Could you also add `'@typescript-eslint/no-unused-vars': 'off',` here as we already did in other PRs

https://github.com/pioardi/poolifier/pull/91/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R35
https://github.com/pioardi/poolifier/pull/92/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R35

Or maybe we should merge https://github.com/pioardi/poolifier/pull/91 so we don't need this in every single open PR